src/components/ebay-page-notice/test/test.server.js

```diff
@@ -10,7 +10,7 @@ describe('page-notice', () => {
     it('renders with defaults', async() => {
         const input = mock.Page;
         const { getAllByLabelText, getByText } = await render(template, input);
-        const status = getAllByLabelText(input.a11yText)[1].parentElement;
+        const status = getAllByLabelText(input.a11yText)[0].parentElement;
```

Comment:
This one didn't work with `getByLabelText`?

Reply:
ah dang missed that one. no it should work. one sec